src/transformers/modeling_tf_bart.py

```diff
@@ -262,7 +261,7 @@ def call(self, x, encoder_padding_mask, training=False):
         x = self.activation_fn(self.fc1(x))
         x = self.activation_dropout(x, training=training)
         x = self.fc2(x)
-        x = self.dropout_wt(x, training=training)
+        x = tf.nn.dropout(x, rate=self.dropout if training else 0)
```

Comment:
You're defining two ways of doing the dropout here, can this be consistent?
Shouldn't this just be
```py
x = self.dropout(x, training=training)
```
since you initialized `dropout` on line 229? (i.e., exactly as it was before?)

Reply:
yes, will fix.

Reply:
switched to always using `tf.nn.dropout(x, rate=x if training else 0)`